visualization/app/codeCharta/ui/scenarioDropDown/scenarioDropDown.component.html

```diff
@@ -18,12 +18,17 @@
 				class="button-hovering"
 				ng-click="$ctrl.applyScenario(scenario.scenarioName)"
 				ng-disabled="!scenario.isScenarioAppliable"
+				title="Apply scenario"
 				>{{ scenario.scenarioName }}</md-button
 			>
 			<span class="content-icons" ng-repeat="icon in scenario.icons track by $index">
 				<i class="fa {{ icon.faIconClass }}" ng-class="{ 'is-saved': icon.isSaved }"></i>
```

Comment:
These should likely also get a tooltip. It's unclear what the icons actually stand for.